tasks/karma-tests.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+const spawn = require( "child_process" ).spawn;
+
+module.exports = function( grunt ) {
+
+	// The task runs tests in various browser sets but does it sequentially in
+	// separate processes to avoid Karma bugs that cause browsers from previous
+	// sets to somehow still be waited on during subsequent runs, failing the build.
+	grunt.registerTask( "karma-tests", "Run unit tests sequentially",
+		async function( isBrowserStack ) {
```

Comment:
Can you fix the indentation here?

Reply:
@gibson042 Actually, what would be the correct indentation here?

Reply:
Do you mean that everything inside of this function should be indented one level deeper?

Reply:
Yes, I think so. Either that, or give this line _extra_ indentation to show that it is logically part of the preceding, e.g.
```
	grunt.registerTask( "karma-tests", "Run unit tests sequentially",
			async function( isBrowserStack ) {

		const done = this.async();
		…
	} );
```
(but I think I'd prefer the former)

Reply:
@gibson042 PR updated.